src/test/lib/HandshakeTest.cpp

```diff
@@ -138,6 +147,9 @@ QuicTestConnect(
                 if (!Client.WaitForZeroRttTicket()) {
                     return;
                 }
+                //
+                // TODO - Don't wait for client/server shutdown when using random loss.
```

Comment:
Is this going to be part of this PR? Or a later PR?

Reply:
Later. Possibly.

Reply:
Ended up fixing it anyways.